packages/test-utils/lib/e2e/clean-appointment.ts

```diff
@@ -24,3 +36,186 @@ export const cleanAppointment = async (appointmentId: string, env: string): Prom
     return false;
   }
 };
+
+export const cleanAppointmentGraph = async (tag: Coding, oystehr: Oystehr): Promise<boolean> => {
+  const allResources = await getAppointmentGraphByTag(oystehr, tag);
+  const [deleteRequests, persons] = generateDeleteRequestsAndPerson(allResources);
+
+  await Promise.all(persons.map((person) => patchPerson(oystehr, person, allResources)));
+
+  try {
+    console.log(`Deleting resources...`);
+    const transaction = await oystehr.fhir.transaction({ requests: [...deleteRequests] });
+    console.log(`Transaction response:`, JSON.stringify(transaction, null, 2));
+    return true;
+  } catch (e) {
+    console.log(`Error deleting resources: ${e}`, JSON.stringify(e));
+    return false;
+  }
+};
+
+// these things are important environment infra rather than ephemeral test data, so leave them alone
+const NEVER_DELETE = [
+  'Person',
```

Comment:
the e2e tests definitely flood patient data, wouldn't we want to clear those up too (cautiously)?